vertx-grpc-transcoding/src/main/java/module-info.java

```diff
@@ -0,0 +1,13 @@
+module io.vertx.grpc.transcoding {
+  requires io.netty.common;
+  requires io.netty.buffer;
+  requires io.netty.codec;
+  requires io.netty.codec.compression;
+  requires io.netty.transport;
+  requires com.google.protobuf;
+  requires com.google.protobuf.util;
+  requires transitive io.vertx.core;
```

Comment:
avoid _transitive_ here, for the moment we avoid transitive dependency when declaring module descriptors